lib/cogctl/actions/permissions/view.ex

```diff
@@ -23,13 +23,12 @@ defmodule Cogctl.Actions.Permissions.View do
   end
 
   defp format_table(permissions) when is_list(permissions) do
-    [["BUNDLE", "NAME", "ID"] | Enum.map(permissions, &([&1["bundle"], &1["name"], &1["id"]]))]
+    [["NAME", "ID"] | Enum.map(permissions, &(["#{&1["bundle"]}:#{&1["name"]}", &1["id"]]))]
```

Comment:
Ah, I actually wrote this a while back. We sort all tabled views by row, so this is all sorted just as you'd expect.